pkg/operator/certrotationcontroller/certrotationcontroller.go

```diff
@@ -135,7 +135,7 @@ func newCertRotationController(
 			Name:      "aggregator-client-signer",
```

Comment:
is this the signer used to sign `--proxy-client-cert` ?

Reply:
Yes